pkgs/applications/editors/emacs/sources.nix

```
@@ -4,8 +4,8 @@
 }:
 
 let
-  mkArgs = { pname, version, variant, patches ? _: [ ], rev, hash }: {
-    inherit pname version variant patches;
+  mkArgs = { pname, version, variant, templateNativeCompDriverOptionsPatch, patches ? _: [ ], rev, hash }: {
+    inherit pname version variant templateNativeCompDriverOptionsPatch patches;
```

What about adding `templateNativeCompDriverOptionsPatch` to `patches`?

```suggestion
    inherit pname version variant templateNativeCompDriverOptionsPatch
    patches = patches ++ [ templateNativeCompDriverOptionsPatch ];
```